sound/soc/sof/loader.c

```diff
@@ -530,7 +531,7 @@ int sof_fw_ready(struct snd_sof_dev *sdev, u32 msg_id)
 
 	sof_get_windows(sdev);
 
-	return 0;
+	return  sof_ipc_init_msg_memory(sdev);
```

Comment:
do we need a function for this? Can we simply not init the ipc msg here directly?

Reply:
we can't since all the ipc definition is defined in ipc.c so you can't access it out of ipc.c. I think the idea is to deal with ipc in one file.